### DISCUSSION TITLE
Expired Domain

### DISCUSSION THREAD

**Discussion by nixxxo** [vercel/community#5781] [Help]
### Summary

Hello,
one of my domains: waveme.app expired without me noticing and I really need to renew it, I submitted a case, but I don't see any sort of confirmation or contact back, what can I do to renew my domain.

### Steps to Reproduce

Go to domains and check domains.

**Comment by amyegan** [comment]
Hey @nixxxo. If you let me know the case number, I can check on the ticket for you 🙏

**Reply by nixxxo** [reply]
Case number is: Case (#00196562) created successfully.

**Reply by amyegan** [reply]
Thanks for sharing the case number. I see you have a response on your case now, so I'm going to close this discussion. Please continue to follow up in your support case with any further concerns 🙂
